src/Diagnostics.RuntimeHost/Utilities/DateTimeHelper.cs

```diff
@@ -103,9 +103,9 @@ internal static bool PrepareStartEndTimeUtc(string startTime, string endTime, ou
                 errorMessage = "Invalid Start Time and End Time. End Time cannot be earlier than Start Time.";
                 return false;
             }
-            else if (startTimeUtc > currentUtcTime)
+            else if (startTimeUtc > currentUtcTime.AddMinutes(-15))
```

Comment:
Should we have a similar check for end time too ? UI forces the endtime to be 15 minutes less than the current time.